web/styles/zulip.css

```diff
@@ -1459,6 +1456,24 @@ a.dark_background:hover,
     color: hsl(200deg 99% 60%);
 }
 
+/* UI redesign: blue links style: https://github.com/zulip/zulip/issues/24877 */
+.dark_background a,
+.dark_background a:link,
+.dark_background a:visited {
+    color: hsl(200deg 100% 50% / 100%);
+    text-decoration-color: hsl(200deg 100% 50% / 30%);
+}
+
+.dark_background a:hover {
+    color: hsl(200deg 100% 66% / 100%);
+    text-decoration-color: hsl(200deg 100% 66% / 70%);
+}
+
+.dark_background a:active {
+    color: hsl(200deg 100% 55% / 100%);
+    text-decoration-color: hsl(200deg 100% 50% / 70%);
+}
```

Comment:
Why are these selectors different from the shape of the ones in the main theme?

Or maybe the dark background changes are wrong and you meant to do something in; `web/styles/dark_theme.css`.